### PR TITLE
Reorganize solution

### PR DESCRIPTION
**Pull request by alan-null**
Fixes: #119 Reorganize solution

### CODE REVIEW

File: src/Sitecore.Modules.WeBlog.Mvc/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-[assembly: AssemblyFileVersion("2.5.0.0")]
-
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Sitecore.Modules.WeBlog")]
```
Comment (adeneys) [inline src/Sitecore.Modules.WeBlog.Mvc/Properties/AssemblyInfo.cs]: The assembly title is missing the "MVC" at the end.

File: src/Sitecore.Modules.WeBlog.Mvc/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Sitecore.Module.WeBlog")]
```
Comment (adeneys) [inline src/Sitecore.Modules.WeBlog.Mvc/Properties/AssemblyInfo.cs]: Leave the product name as is. Even though this is the Sitecore.Module.WeBlog.MVC file, it belongs to the Sitecore.Modules.WeBlog product.

File: src/Sitecore.Modules.WeBlog.WebForms/Layouts/WeBlog/Entry.ascx [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
-    <% } %>
-    <% if(DoesFieldRequireWrapping("Content")) { %>
-        <p>
-    <% } %>
-    <sc:Text ID="txtContent" Field="Content" runat="server" />
-    <% if(DoesFieldRequireWrapping("Content")) { %>
-        </p>
-    <% } %>
-    <sc:Placeholder runat="server" key="weblog-below-entry" /> 
+﻿<%@ Control Language="C#" AutoEventWireup="true" CodeBehind="Entry.ascx.cs" Inherits="Sitecore.Modules.WeBlog.WebForms.Layouts.WeBlog.BlogEntry" %>
```
Comment (adeneys) [inline src/Sitecore.Modules.WeBlog.WebForms/Layouts/WeBlog/Entry.ascx]: Update the namespace for sublayouts so we don't repeat the "WeBlog" name. I know this goes against VS default naming, but it's more succinct. Please move all sublayout codebehinds to the Sitecre.Modules.WeBlog.WebForms.Layouts namespace

File: src/Sitecore.Modules.WeBlog.WebForms/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Sitecore.Modules.WeBlog")]
```
Comment (adeneys) [inline src/Sitecore.Modules.WeBlog.WebForms/Properties/AssemblyInfo.cs]: As with the MVC assembly info file, make sure you've got "WebForms" in the assembly title.

File: Sitecore.Modules.WeBlog.sln [hunk 5]
```diff
@@ -5,10 +5,14 @@ VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "IntegrationTest", "IntegrationTest\IntegrationTest.csproj", "{DD0793C1-00C7-48A1-BCEA-28FB08BC9CD8}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sitecore.Modules.WeBlog", "Website\Sitecore.Modules.WeBlog.csproj", "{26FE5F3A-F140-40C4-85A0-C1866DA26F16}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sitecore.Modules.WeBlog", "src\Sitecore.Modules.WeBlog\Sitecore.Modules.WeBlog.csproj", "{26FE5F3A-F140-40C4-85A0-C1866DA26F16}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTest", "UnitTest\UnitTest.csproj", "{9289295B-542B-4A8A-A951-40DC9F5BC2D0}"
```
Comment (adeneys) [inline Sitecore.Modules.WeBlog.sln]: Let's move all source under the new "src" folder, including the unit test and integration test projects. I'd also move the Themes folder under src as it is source.

**Comment by alan-null** [comment]
@adeneys Thanks

**Pull request merged**